### PR TITLE
make sure the Job CLI support multi-config formats

### PR DESCRIPTION
**Pull request by chesterxgchen**
Fixes # .

Fix few cases where the Job CLI only handle one specific configuration format

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/utils/cli_utils.py [hunk 1]
```diff
@@ -160,26 +161,57 @@ def is_dir_empty(path: str):
...
-    fmt = ConfigFormat.JSON if to_json else ConfigFormat.PYHOCON
-    ext = ConfigFormat.extensions(fmt)[0]
-    if dst_path.endswith(ext):
+def hocon_to_string(target_fmt: ConfigFormat, dst_config: ConfigTree):
+    if target_fmt == ConfigFormat.JSON:
+        return HOCONConverter.to_json(dst_config)
+    elif target_fmt == ConfigFormat.PYHOCON:
+        return HOCONConverter.to_hocon(dst_config)
+    elif target_fmt == ConfigFormat.OMEGACONF:
+        from nvflare.fuel_opt.utils.omegaconf_loader import OmegaConfLoader
```
Comment (IsaacYangSLA) [inline nvflare/utils/cli_utils.py:170]: Not sure if the import statement should be here.  The PyhoconConfig is at the top.

**Review by IsaacYangSLA — APPROVED** [review]
LGTM.  This PR mainly adds support of OMEGACONF.  I think the OmegaConfLoader import location is OK, but for consistency, it should be at the top.

File: nvflare/utils/cli_utils.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 from pyhocon import ConfigTree, HOCONConverter
```
Comment (YuanTingHsieh) [inline nvflare/utils/cli_utils.py:20]: not restricted to this PR.
We have all these "utils" file all over the place and tools/cli is all over the place, maybe we want to clean them up later.

**Review by YuanTingHsieh — APPROVED** [review]
Some other old tools might need to be updated as well so they can support multiple config formats.

**Pull request merged**